### ISSUE TITLE
the role doesn't work like the example

### ISSUE THREAD

**Issue by mcocera** [morbidick/ansible-role-nfs#1]
Hi, 
I just testing the role an i've encountered a little problem, i don't know if i do something wrong or really doesn't work fine. 
 If declare

> - nfs_exports:
      - name: "share"
        path: "/srv/share"
        subnet: "192.168.1.100/32"
        writable: yes

 the result on the /etc/exports it's the following: 

> {u'writable': True, u'path': u'/data/ta', u'name': u'share', u'subnet': u'192.168.1.100/32'}

not the desired: 

> /srv/share 192.168.1.100(writable)

I don't know if i makes well the procedure or it's a problem with the template or any thing of the role.

**Comment by morbidick** [comment]
thx for reporting this, at first glance the only thing that could have happened is a wrong indentation level?  Could you try it with the following snippet:

```yaml
nfs_exports:
  - name: "share"
    path: "/srv/share"
    subnet: "192.168.1.100/32"
    writable: yes
```
ill try to reproduce later today.

**Comment by mcocera** [comment]
I revised the indentation levels and they're ok
I found the error, it's on the template, when install the role from ansible-galaxy, the template it's different like github. On my downloaded template looks like this:

> {% for export in nfs_exports %}
{{ export }}
{% endfor %}

possibly needs update on ansible-galaxy?

**Comment by morbidick** [comment]
thx for dissecting this, this is probably the reason i use git urls directly and didnt encounter this issue. I published v1.3.0 and released it to galaxy.
